src/dscanner/analysis/lambda_return_check.d

```diff
@@ -52,15 +51,45 @@ extern (C++) class LambdaReturnCheck(AST) : BaseAnalyzerDmd
 		if (lambda.fbody.isReturnStatement() is null)
 			return;
 
-		auto tokenRange = tokens.filter!(t => t.loc.fileOffset > lambda.loc.fileOffset)
-			.filter!(t => t.loc.fileOffset < lambda.endloc.fileOffset)
-			.find!(t => t.value == TOK.goesTo);
+		auto lambdaRange = tokens.filter!(t => t.loc.fileOffset > lambda.loc.fileOffset)
+			.filter!(t => t.loc.fileOffset < lambda.endloc.fileOffset);
+		auto tokenRange = lambdaRange.find!(t => t.value == TOK.goesTo);
```

Comment:
This change is unrelated to what you are trying to achieve. Please keep the diff at minimum.

Reply:
It is related. For detecting the warning, only the token range starting from the arrow symbol is necessary. For Autofix, the token range must start from the beginning of the lambda expression.